recipe/build.sh

```diff
@@ -1,14 +1,18 @@
 #!/bin/sh
+set -ex
 
 ./bootstrap \
+             --verbose \
              --prefix="${PREFIX}" \
              --system-libs \
+             --system-libuv \
              --no-qt-gui \
              --no-system-libarchive \
              --no-system-jsoncpp \
              --parallel=${CPU_COUNT} \
              -- \
              -DCMAKE_BUILD_TYPE:STRING=Release \
-             -DCMAKE_FIND_ROOT_PATH="${PREFIX}" \
```

Comment:
You should keep this. Directory specified here will be searched before system directories

Reply:
good catch! I wonder how those got deleted